eng/pipelines/coreclr/templates/build-perf-maui-apps.yml

```diff
@@ -129,8 +129,9 @@ steps:
 
   - script: |
       chmod -R a+r .
-      ../dotnet publish -bl:MauiAndroid.binlog -f net6.0-android -c Release
-      mv ./bin/Release/net6.0-android/com.companyname.MauiTesting-Signed.apk ./MauiAndroidDefault.apk
+      ../dotnet restore
```

Comment:
Can you add a comment with a link to the issue we are working around here?

Reply:
For this one, https://github.com/dotnet/sdk/issues/21877.

Reply:
Can you add a comment in the code with a link to the issue we are working around here?

Reply:
Oh, yes